prosody/rootfs/defaults/saslauthd.conf

```diff
@@ -1,7 +1,7 @@
 {{ if eq (.Env.AUTH_TYPE | default "internal") "ldap" }}
 ldap_servers: {{ .Env.LDAP_URL }}
 ldap_search_base: {{ .Env.LDAP_BASE }}
-{{ if .Env.LDAP_BINDDN | default "" | toBool }}
+{{ if .Env.LDAP_BINDDN | default "" }}
```

Comment:
Do we really need the default value?

Reply:
I think adding the default value was the entire point of #238.

Reply:
I was trying to understand the problem but was not able to figure it out. The following commands are returning nothing so I can't see the problem with `<no value>`.
`LDAP_BINDDN` is unset on my machine.

```bash
echo "{{ if .Env.LDAP_BINDDN | default "" }}ldap_bind_dn: {{ .Env.LDAP_BINDDN }}{{ end }}" | frep -
```

```bash
echo "{{ if .Env.LDAP_BINDDN }}ldap_bind_dn: {{ .Env.LDAP_BINDDN }}{{ end }}" | frep -
```

Reply:
I see. So maybe #238 and therefore this PR are entirely useless?

Reply:
It seems so or I just don't understand the problem.